.travis.yml

```diff
@@ -30,11 +31,14 @@ branches:
   only:
   - master
 
+# Skipping the install step or otherwise it would trigger enforcer plugin and fail.
+install: true
```

Comment:
I can update the comment to be more precise. The main reason is that we actually do not need the install step at all, ac the script step will do the same, i.e. with install step we would just run the maven twice.